qa/rpc-tests/fundrawtransaction.py

```diff
@@ -677,6 +677,14 @@ def run_test(self):
         assert(signedtx["complete"])
         self.nodes[0].sendrawtransaction(signedtx["hex"])
 
+        inputs = []
+        outputs = {self.nodes[2].getnewaddress() : 1}
+        rawtx = self.nodes[3].createrawtransaction(inputs, outputs)
+        result = self.nodes[3].fundrawtransaction(rawtx, )
```

Comment:
Nit: empty arg.
Nit: Missing comment mentioning that `min_relay_tx_fee=1000` is used.